diesel_derives/src/queryable_by_name.rs

```diff
@@ -118,3 +121,38 @@ fn sql_type(field: &Field, model: &Model) -> syn::Type {
         }
     }
 }
+
+fn check_sql_type(field_ty: &syn::Type, st: &syn::Type) {
+    // syn heapsize example trick won't work here because there is an additional database generic
+    // unless we create another trait without the DB part
+    //
+    //     quote_spanned! {field_ty.span()=>
+    //         function(st)
```

Comment:
```suggestion
    // syn heapsize example trick won't work here because there is an additional database generic
    // unless we create another trait without the DB part.
    //
    //     quote_spanned! { field_ty.span() =>
    //         function(st)
```